client/app/templates/application.hbs

```diff
@@ -1,23 +1,25 @@
-<LabsUi::SiteHeader
-  @responsiveNav={{true}}
-  @responsiveSize="medium"
-  @betaNotice={{true}}
-  as |banner|
->
-  <banner.title>
-    <LinkTo @route="application" class="site-title">
-      Applicants
-      <small class="site-subtitle show-for-medium">File and track your applications</small>
-    </LinkTo>
-  </banner.title>
-  <banner.nav>
-    <ul class="menu vertical medium-horizontal">
-      {{#if this.session.isAuthenticated}}
-        <Auth::UserBadge />
-      {{/if}}
-    </ul>
-  </banner.nav>
-</LabsUi::SiteHeader>
+{{#if this.header}}
```

Comment:
Is this necessary for this PR? 

It looks like there were no changes to this file other than adding this conditional -- the rest is reformatting, is that right?